data/users.yml

```diff
@@ -2148,3 +2148,9 @@ jwbensley:
   rir_handle: JBIL-RIPE
   companies:
   - as5405
+vincentz:
+  companies:
+     - as55064
```

Comment:
do you want to add `Florian` as well?

Reply:
https://github.com/yeled/networker/blob/02d4716dd37d9c8fed55b702b17be0a4d0f772c2/data/users.yml#L1110

Reply:
Yes, good point. Will do that 😌.